packages/talos_suffix/src/utils.rs

```diff
@@ -0,0 +1,8 @@
+/// Retrives all the items that are valid and not None.
+pub fn get_nonempty_suffix_items<'a, T: 'a>(items: impl Iterator<Item = &'a Option<T>> + 'a) -> impl Iterator<Item = &'a T> + 'a {
```

Comment:
Test can be written for these without the context of suffix

Reply:
updated in latest commit